test/producer-consumer-spec.js

```diff
@@ -141,4 +141,17 @@ describe('producer/consumer', function () {
       });
     });
   });
+
+  describe('undefined queue name', () => {
+    before(() => {
+      bunnymq.connection._config.consumerSuffix = undefined;
+    });
+
+    it('should receive message', () => {
+      bunnymq.consume('queue-name-undefined-suffix', (message) => {
+        assert(message.msg === 'test for undefined queue suffix');
```

Comment:
Your way it is ok, but in such case of a comparison, ``assert.equal`` will fit better. And it will also give use a better AssertionError message (in case it occurs)